ViewModel/ImpactAreaScenario/Editor/SpecificIASEditorVM.cs

```diff
@@ -335,7 +335,16 @@ private void UpdateRatingRequired()
         {
             if (SelectedFrequencyElement != null && SelectedFrequencyElement.ChildElement is AnalyticalFrequencyElement elem)
             {
-                RatingRequired = !elem.IsAnalytical;
+                RatingRequired = false;
+                if(elem.IsAnalytical)
```

Comment:
I recommend RatingRequired = elem.IsAnalytical || elem.MyGraphicalVM.UseFlow or collapsing the if block.

Reply:
This is hard to do, it is as simple as your suggestion because MyGraphicalVM always has values. Bottom line is i think it is simpler to read the way that it is.